src/test/java/org/hl7/davinci/priorauth/ClaimSubmitTest.java

```diff
@@ -27,13 +29,15 @@
 import okhttp3.RequestBody;
 import okhttp3.Response;
 
-@RunWith(MonoMeecrowave.Runner.class)
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
```

Comment:
Cool.   I didn't know you could generate a random port!

Reply:
Neither did I! But it means we can now run the tests without stopping the service!